examples/react/view-transitions/src/main.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import { RouterProvider, createRouter } from '@tanstack/react-router'
+import { routeTree } from './routeTree.gen'
+import './styles.css'
+
+// Set up a Router instance
+const router = createRouter({
+  routeTree,
+  defaultPreload: 'intent',
+  defaultStaleTime: 5000,
+  scrollRestoration: true,
```

Comment:
Include a comment stating that using `defaultViewTransition` would make it so that the user wouldn't have to add `viewTransition: true` to every navigation call.